data/data/powervs/cluster/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "cloud_instance_id" {
```

Comment:
this can be deleted and just use the output from the iaas stage after you put it into data/data/powervs/variables-powervs.tf